README.md

```diff
@@ -140,9 +140,9 @@ await fastify.register(import('@fastify/rate-limit'), {
 })
 ```
 
-- `global` : indicates if the plugin should apply the rate limit setting to all routes within the encapsulation scope
-- `max`: is the maximum number of requests a single client can perform inside a timeWindow. It can be an async function with the signature `async (request, key) => {}` where `request` is the Fastify request object and `key` is the value generated by the `keyGenerator`. The function **must** return a number.
-- `ban`: is the maximum number of 429 responses to return to a single client before returning 403. When the ban limit is exceeded the context field will have `ban=true` in the errorResponseBuilder. This parameter is an in-memory counter and could not work properly in a distributed environment.
+- `global` : indicates if the plugin should apply rate limiting to all routes within the encapsulation scope.
+- `max`: maximum number of requests a single client can perform inside a timeWindow. It can be an async function with the signature `async (request, key) => {}` where `request` is the Fastify request object and `key` is the value generated by the `keyGenerator`. The function **must** return a number.
+- `ban`: maximum number of 429 responses to return to a client before returning 403 responses. When the ban limit is exceeded, the context argument that is passed to `errorResponseBuilder` will have its `ban` property set to `true`. This parameter is an in-memory counter and will not work properly in a distributed environment.  **Note:** `0` can also be passed to directly return 403 responses when a client exceeds the limit.
```

Comment:
regarding the last sentence:

0 is the limit ("exceeds the limit"), and returning 403 response is the effect. I would prefer to write that 0 is a valid value and results in an immediatly ban and that the server will return a 403. 

Maybe @Fdawgs has some suggestions